aptoide-installer/src/main/java/cm/aptoide/pt/installer/AptoideInstaller.kt

```diff
@@ -55,6 +56,10 @@ class AptoideInstaller @Inject constructor(
         android.content.pm.PackageInstaller
           .SessionParams(android.content.pm.PackageInstaller.SessionParams.MODE_FULL_INSTALL)
           .apply {
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.UPSIDE_DOWN_CAKE) {
+              setRequestUpdateOwnership(true)
+              setRequireUserAction(USER_ACTION_NOT_REQUIRED)
```

Comment:
The USER_ACTION_NOT_REQUIRED flag was actually added on api level 31, contrarily to the update ownership which was only added in app level 34. Extract the user action flag to a separate condition check for api level 31